src/pages/selectService/selectService.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Service from "@components/customer/selectService/serviceFather.tsx";
+import LayoutSelectService from "@layout/customer/selectService/layoutSelectService.tsx";
+import "../../css/customer/selectService/selectService.css";
+
+import "@fontsource/roboto/100.css";
+import "@fontsource/roboto/300.css";
+import "@fontsource/roboto/400.css";
+import "@fontsource/roboto/500.css";
+import "@fontsource/roboto/700.css";
+import "@fontsource/roboto/900.css";
```

Comment:
are these imports necessary?

Reply:
For when you want to use the font in the future, since it is not a variable font, it is better to export it all to avoid future errors.